.github/workflows/commit-message-checker.yml

```
@@ -0,0 +1,19 @@
+name: 'Commit Message Checker'
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
```

What about?
```suggestion
on: [pull_request]
```
As described in their documentation: https://github.com/wagoid/commitlint-github-action#usage